### ISSUE TITLE
[Feature Request] Floating taskbar

### ISSUE THREAD

**Issue by FelixSchuetze** [TranslucentTB/TranslucentTB#423]
[x] I have checked the issue tracker if my question or bug has
already been fixed or answered in another (possibly closed)
issue.

I would really love to have my taskbar floating, with rounded corners and variable length. Is there a possibility of seeing this in a future update?

**Comment by sylveon** [comment]
@torchgm made a tool to do this: https://github.com/torchgm/RoundedTB/releases/tag/P3

It can be used in conjunction of TTB as well.

**Comment by torchgm** [comment]
(FWIW RoundedTB isn't finished yet, especially not the feature you're looking for. I'm working closely with Sylly to make sure that TTB compatibility works properly as unfortunately Windows' DWM causes a couple of annoying issues [which are probably my fault sorry])

TL;DR I'm working on it, promise!

**Comment by Martiks** [comment]
![Screenshot 2021-11-11 004525](https://user-images.githubusercontent.com/52933808/141211493-46c7f754-1cb8-4910-bd85-b89ff817e0f0.png)
I would love an option to have programs maximized with the taskbar being on top like this
![image](https://user-images.githubusercontent.com/52933808/141211630-f50d354a-4c53-45d6-8882-522414bdf839.png)
instead of normal like now
